development-tools/clinic.rst

```diff
@@ -2000,52 +2000,75 @@ The generated glue code looks like this:
 .. versionadded:: 3.13
 
 
-.. _clinic-howto-getter:
+.. _clinic-howto-pygetsetdef:
 
-How to generate a getter
-------------------------
+How to declare PyGetSetDef functions
+------------------------------------
 
-"Getters" are C functions that facilitate property-like access for a class.
-See :c:type:`getter <PyGetSetDef>` for details.
-You can use the ``@getter`` directive to generate an "impl" function for a
-getter using Argument Clinic.
+"Getters" and "setters" are C functions that facilitate property-like access for a class.
+See :c:type:`getter <PyGetSetDef>` and :c:type:`setter <PyGetSetDef>` for details.
+You can use the ``@getter`` and ``@setter`` directives to generate
+"impl" functions for getters and setters in Argument Clinic.
```

Comment:
Hm, I think we can make this more succinct:

- we don't need to repeat "getters and setters" again; we already said that three times in the preceding lines
- "in Argument Clinic" is implicit; I'd say drop it :)

```suggestion
"impl" functions for these.
```

What do you think?

Reply:
Looks more simpler!

Reply:
Oh, I actually quite like appending "using Argument Clinic" to this paragraph -- while it's true the whole document is about Argument Clinic, people won't necessarily have read the preceding sections before jumping to this section, and this paragraph is introductory to this subtopic

Reply:
Added!